.github/workflows/ci.yml

```diff
@@ -10,15 +10,16 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Linelint
         uses: fernandrone/linelint@master
       - name: Set up Node
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v3
         with:
           node-version: 16.14.2
+          cache: npm
       - name: Install dependencies
-        run: npm install
+        run: npm ci
```

Comment:
According to the [official doc](https://docs.npmjs.com/cli/v6/commands/npm-ci)

> This command is similar to [npm install](https://docs.npmjs.com/cli/v6/commands/npm-install), except it's meant to be used in automated environments such as test platforms, continuous integration, and deployment -- or any situation where you want to make sure you're doing a clean install of your dependencies. It can be significantly faster than a regular npm install by skipping certain user-oriented features.